### PR TITLE
Enhance Time table Feature

### LINKED ISSUE #144: Add timetable info in the application
**Is your feature request related to a problem? Please describe.**
Now that we have the backend for timetables, we can integrate into the application.
Add a new module which fetches the timetable of the user's branch and year and displays it in a nice UI.

### PR DESCRIPTION
**Pull request by sheetal-g9**
closes:#144

Changes:  Enhance the timetable layout making it compatible to be viewed in both orientations .
                 Fix the timetable selector dialog to display Title and button. 

Screenshots for the change:
timetable_dialog_layout
timetable_layout

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ task clean(type: Delete) {
 
 ext {
     // sdk and tools
-    minSdkVersion = 16
+    minSdkVersion = 21
```
Comment (dr0pdb) [inline build.gradle]: Why do you always make unnecessary changes?

**Review by dr0pdb — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (BVShe001) [reply build.gradle]: that was required for some  xml attributes

File: app/src/main/java/com/iitp/njack/iitp_connect/core/timetable/TimeTableActivity.java [hunk 2]
```diff
@@ -24,7 +23,7 @@
 
 import javax.inject.Inject;
 
-public class TimeTableActivity extends AppCompatActivity implements View.OnClickListener {
+public class TimeTableActivity extends AppCompatActivity{
```
Comment (ravi-kishan) [inline app/src/main/java/com/iitp/njack/iitp_connect/core/timetable/TimeTableActivity.java]: Add space before brace

File: app/src/main/java/com/iitp/njack/iitp_connect/core/timetable/TimeTableActivity.java [hunk 3]
```diff
@@ -91,32 +90,32 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
             timetableDialogBinding.timeTableBranchChange.setAdapter(branchAdapter);
             timetableDialogBinding.timeTableYearChange.setAdapter(yearAdapter);
+            builder.setPositiveButton("SUBMIT", (dialog, which) -> {
+                timeTableInformation.setCourse(timetableDialogBinding.timeTableCourseChange.getSelectedItem().toString());
+                timeTableInformation.setBranch(timetableDialogBinding.timeTableBranchChange.getSelectedItem().toString());
+                timeTableInformation.setYear(timetableDialogBinding.timeTableYearChange.getSelectedItem().toString());
+                dialog.dismiss();
+                setValues();
+            });
+            dialog=builder.create();
```
Comment (ravi-kishan) [inline app/src/main/java/com/iitp/njack/iitp_connect/core/timetable/TimeTableActivity.java]: Add space before and after =

**Review by ravi-kishan — APPROVED** [review]
The PR improves timetable feature

**Review by dr0pdb — APPROVED** [review]
